ksp_plugin/frames.hpp

```diff
@@ -84,6 +84,18 @@ using Camera = Frame<serialization::Frame::PluginTag,
 using RigidPart = Frame<serialization::Frame::PluginTag,
                         serialization::Frame::RIGID_PART, false>;
 
+// The |PileUp| is seen as a (currently non-rotating) rigid body; the degrees of
+// freedom of the parts in the frame of that body can be set, however their
+// motion is not integrated; this is simply applied as an offset from the rigid
+// body motion of the |PileUp|.
+// The origin of |RigidPileUp| is the centre of mass of the pile up.  Its axes
+// are those of Barycentric for now; eventually we will probably want to use the
+// inertia ellipsoid.
```

Comment:
I don't think we will want to use the principal axes of the pile up here, since we still want things to work out of timewarp, where these axes could vary discontinuously (but as you say, this comment will need to change anyway).

Reply:
I think we'll want both, but we'll see.